newspaper/images.py

```diff
@@ -25,6 +27,8 @@
 thumbnail_size = 90, 90
 minimal_area = 5000
 
+BAD_URLS = ['logo', 'icon', 'banner', 'ribbon', 'sprite', 'qrcode']
```

Comment:
This is based on my own research, I tried to be conservative to avoid stripping out interesting images. I could see making this configurable but it is not currently on the other valid_url checks